pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/utilities/InputData.java

```diff
@@ -46,6 +46,7 @@
     protected String accessor;
     protected String resolver;
     protected String fragmenter;
+    protected String metadata;
```

Comment:
since this is actually the MetadataFetcher class name, perhaps rename metadataFetcher?